src/libraries/System.Private.CoreLib/src/System/Buffers/StandardFormat.cs

```diff
@@ -182,7 +182,8 @@ internal int Format(Span<char> destination)
 
                     if (precision >= 10)
                     {
-                        uint div = Math.DivRem(precision, 10, out precision);
+                        uint div;
+                        (div, precision) = Math.DivRem(precision, 10);
```

Comment:
Is there an advantage using the new overloads here and similar places below?  Does the codegen end up being better?

Reply:
@stephentoub sorry for the delay, didn't notice notifications.

I couldn't find any differences in terms of codegen, this and other changes are made to remove internal DivRem functions